### PR TITLE
chore: move EIP-4626 to Last Call

### PR DESCRIPTION
**Pull request by fubuloubu**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Review by lightclient — CHANGES_REQUESTED** [review]
Hi, there are 2 instances of "ERC20". They should be replaced with ERC-20. Otherwise, it looks good.

**Comment by axic** [comment]
Please note there are some pending PRs which I think should be merged prior to going to last call.

**Comment by sambacha** [comment]
> @axic, thanks for pointing that out. It appears there are two outstanding PRs:
> 
> * [4626: clarify behavior #4777](https://github.com/ethereum/EIPs/pull/4777)
> * [4626: Some copywriting changes #4781](https://github.com/ethereum/EIPs/pull/4781)
> 
> @fubuloubu, could you address these PRs and resolve the conflicts in this one? Thanks.

https://github.com/ethereum/EIPs/pull/4806 included all of the changes in this PR. This can close

_Originally posted by @Joeysantoro in https://github.com/ethereum/EIPs/issues/4781#issuecomment-1039722848_

seems like we are ready for last call now
💯

**Comment by MicahZoltu** [comment]
Needs conflicts resolved.

**Comment by fubuloubu** [comment]
> Needs conflicts resolved.

resolved

**Comment by Joeysantoro** [comment]
Should be ready to go! @lightclient @MicahZoltu

File: EIPS/eip-4626.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Tokenized Vault Standard
 description: A standard for tokenized Vaults with a single underlying ERC-20 token.
 author: Joey Santoro (@joeysantoro), t11s (@transmissions11), Jet Jadeja (@JetJadeja), Alberto Cuesta Cañada (@alcueca), Señor Doggo (@fubuloubu)
 discussions-to: https://ethereum-magicians.org/t/eip-4626-yield-bearing-vault-standard/7900
-status: Review
+status: Last Call
```
Comment (wschwab) [inline EIPS/eip-4626.md:7]:
```suggestion
status: Last Call
last-call-deadline: 2022-03-08
```

**Review by wschwab — CHANGES_REQUESTED** [review]
this just needs the deadline field for the Last Call deadline, I've added it as a suggestion with two weeks from today, if there are more delays before this gets merged, you might want to move it a bit

File: EIPS/eip-4626.md (hunk 1 above)
Comment (fubuloubu) [reply EIPS/eip-4626.md:7]:
```suggestion
status: Last Call
last-call-deadline: 2022-03-11
```

**Comment by fubuloubu** [comment]
Ready for merging

**Review by wschwab — APPROVED** [review]

**Comment by sambacha** [comment]
> Ready for merging

ser @lightclient bot is blocking pending your approval

**Review by lightclient — APPROVED** [review]
LGTM!

**Pull request merged**